icubam/predicu/data/icu_name_to_department.json

```diff
@@ -1 +1 @@
ienne-CHU": "Loire", "Epinal": "Vosges", "Montbrison-CH": "Loire", "Montargis": "Loiret", "C-Bezannes": "Marne", "Essey": "Meurthe-et-Moselle", "Saverne": "Bas-Rhin", "Saint-Lo": "Manche", "Vannes": "Morbihan", "HPM-Schuman": "Moselle", "Colmar-NC": "Haut-Rhin", "Clermont-MIR": "Puy-de-Dome", "Claude-B": "Moselle", "Roanne-CH": "Loire", "Poitiers-Chir": "Vienne", "Brest-HIA": "Finistere", "HTP-Chir": "Bas-Rhin", "Rennes-CHU": "Ille-et-vilaine", "Orleans-R-Chir": "Loiret", "Nancy-CCV": "Meurthe-et-Moselle", "Caen-CHU": "Calvados", "Poitiers-MIR": "Vienne", "Mont-de-Marsan": "Landes", "Tours-MIR": "Indre-et-Loire", "Angouleme": "Charente", "C-Schweitzer": "Haut-Rhin", "Orleans-MIR": "Loiret", "Charleville-Mezieres": "Ardennes", "Lorient": "Morbihan", "Sarreguemine": "Moselle", "Cherbourg": "Manche", "Saint-Et-C-Mutualiste": "Loire", "Blois": "Loir-et-Cher", "Libourne": "Gironde", "BDX-MIR-HSA": "Gironde", "NHC-Med": "Bas-Rhin", "BDX-PELL-MIR-2": "Gironde", "Nancy-RCP": "Meurthe-et-Moselle", "BDX-PELL-MIR-1": "Gironde", "Nancy-ChirB": "Meurthe-et-Moselle", "St-Dizier": "Marne", "HTP-Med": "Bas-Rhin", "Mulhouse-SSPI": "Haut-Rhin", "Rouen-MIR": "Seine-Maritime", "C-Orangerie": "Bas-Rhin", "Mulhouse-MIR": "Haut-Rhin", "Pau": "Pyr\u00e9nn\u00e9es Atlantique", "SC-OLORON": "Pyr\u00e9nn\u00e9es Atlantique", "Chalons-en-Champagne": "Marne", "Haguenau": "Bas-Rhin", "BDX-C-NORD": "Gironde", "Colmar-Med": "Haut-Rhin", "Verdun": "Meuse", "Nancy-MIRB": "Meurthe-et-Moselle", "CHR-Thionville": "Moselle", "Amiens-MIR": "Somme", "Chateauroux": "Indre", "Morlaix": "Finistere", "Grenoble-MIR": "Isere", "Chateau-Thierry": "Marne", "Dax": "Landes", "Nancy-MIRC": "Meurthe-et-Moselle", "Tours-R-Chir": "Indre-et-Loire", "Mulhouse-Chir": "Haut-Rhin", "Saint-Brieuc": "Cotes-d'armor", "Saint-Malo": "Ille-et-vilaine", "SC-CLIN-NAVARRE": "Pyr\u00e9nn\u00e9es Atlantique", "HTP-Ped": "Bas-Rhin", "Dreux": "Eure-et-Loir", "St-Avold": "Moselle", "Bourges": "Cher", "Tours-R-NChir": "Indre-et-Loire", "Avranches": "Manche", "BDX-PELL-SC-3": "Gironde", "Annonay-CH": "Loire", "BDX-REA-GHS": "Gironde", "Brest-RC": "Finistere", "Brest-MIR": "Finistere", "Saint-Et-Hop-Prive-Loire": "Loire", "C-Rhena": "Bas-Rhin", "Nancy-Ped": "Meurthe-et-Moselle", "Quimper": "Finistere", "Troyes": "Aube", "NHC-ChirC": "Bas-Rhin", "CHR-Mercy": "Moselle", "Grenoble-AR": "Isere", "Bicetre": 93, "A. Beclere": 92, "A. Pare": 93, "Avicenne": 93, "Beaujon": 93, "Bayonne": "Pyr\u00e9nn\u00e9es Atlantique", "Bayonne USC": "Pyr\u00e9nn\u00e9es Atlantique", "BDX R PICQUE": "Gironde", "Alencon": "Orne", "Libourne USC": "Gironde", "Mont de Marsan USC": "Landes", "BDX St AUG": "Gironde", "Perigueux": "Dordogne", "Dax USC": "Landes", "Nantes-MIR": "Loire-Atlantique", "Agen": "Lot et Garonne", "Flers": "Orne", "BDX MIR HSA USC": "Gironde", "Niort": "Deux-Sevre", "a": "Marne", "c": "Marne", "b": "Marne", "d": "Bas-Rhin", "e": "Haut-Rhin", "Agen USC": "Lot et Garonne", "Angers-MIR": "Maine-et-Loire", "Lyon-MIR-CR": "Rhone"}
```

Comment:
Why does this file keep changing? Is it generated by one of the load functions? Should we exclude it from git and let it be re-generated locally as needed?

Reply:
i don t even get why we do not get those from the database to begin with...

Reply:
we do, and actually this file is updated from the database. this is due to some pre-ICUBAM data preprocessing but @dulacarnold and I are currently working on integrating this data to ICUBAM and we won't have to use this file afterwards... i know this looks weird but it was necessary. i will probably remove it in this PR

Reply:
today, @dulacarnold and i pushed the pre-ICUBAM data to the prod's database. i was able to remove this file.